iModelCore/ECDb/ECDb/SchemaWriter.cpp

```diff
@@ -134,6 +134,150 @@ DropSchemaResult SchemaWriter::DropSchema(Utf8StringCR schemaName, SchemaImportC
     }
     return DropSchemaResult(DropSchemaResult::Status::Success);
 }
+
+DropSchemaResult SchemaWriter::DropSchemas(T_Utf8StringVectorCR schemaNames, SchemaImportContext& ctx, bool logIssue)
+    {

```

Comment:
it concerns me that we have implementations for DropSchema and DropSchemas ... couldn't DropSchema be modified to call DropSchemas?

Reply:
Done